runtime/catalog-schema-registry/src/main/java/io/aklivity/zilla/runtime/catalog/schema/registry/internal/handler/SchemaRegistryCatalogHandler.java

```diff
@@ -42,6 +42,7 @@
 public class SchemaRegistryCatalogHandler implements CatalogHandler
 {
     private static final String SUBJECT_VERSION_PATH = "/subjects/{0}/versions/{1}";
+    private static final String SUBJECT_PATH = "subjects/{0}/versions";
```

Comment:
Missing leading slash?

Reply:
Nope, no slash required. Spec is also correct `accept "http://localhost:8081/subjects/items-snapshots-value/versions"`

Reply:
Why does `SUBJECT_VERSION_PATH` start with `/subjects/...` but `SUBJECT_PATH` starts with `subjects/...`?
Seems like these should be consistent, and paths tend to start with `/`, no?

Reply:
oh I see what you mean I got it wrong yep :)